src/lib.rs

```diff
@@ -1,12 +1,16 @@
 // Copyright (c) 2023 The MobileCoin Foundation
 
 #![doc = include_str!("../README.md")]
```

Comment:
:question: Why did the `deny` attribute go away?
```suggestion
#![doc = include_str!("../README.md")]
#![deny(missing_docs, missing_debug_implementations, unsafe_code)]
```

Reply:
It must have got lost when the file was over written. I'll add this.